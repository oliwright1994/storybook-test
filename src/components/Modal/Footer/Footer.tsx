import { ModalFooter as ChakraModalFooter } from '@chakra-ui/core'
import React, { FC } from 'react'

const Footer: FC = ({ children }) => {
  return <ChakraModalFooter>{children}</ChakraModalFooter>
}

export default Footer
