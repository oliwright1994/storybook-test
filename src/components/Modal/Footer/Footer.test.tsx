import React from 'react'
import { renderWithBrightTheme } from 'testing/helpers/themeHelper'

import Footer from '.'

describe('<Footer />', () => {
  const defaultText = 'Test Text'
  it('should render as expected', () => {
    const { container } = renderWithBrightTheme(<Footer>{defaultText}</Footer>)

    expect(container).toMatchSnapshot()
  })

  it('should render passed children in footer tag', () => {
    const { getByText } = renderWithBrightTheme(<Footer>{defaultText}</Footer>)
    const footerEl = getByText(/test text/i)
    expect(footerEl).toBeTruthy()
    expect(footerEl.nodeName).toMatch(/footer/i)
  })
})
