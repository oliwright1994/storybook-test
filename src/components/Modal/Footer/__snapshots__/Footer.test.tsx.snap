// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render as expected 1`] = `
<div>
  <footer
    class="css-81g6me"
  >
    Test Text
  </footer>
</div>
`;
